scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -918,15 +918,15 @@ object a {
 }
 >>>
 object a {
-  def foo: Int = {
+  def foo: Int       = {
```

Comment:
Should we have additional align here and in the two examples below?

If I understand the intention, we should not align if the statements are not just below one another. That's what seems to be happening in `#2750 4 blocks, no blanks` test case.

Reply:
one of things highlighted in the submitted issue is the inconsistent handling of multiline definitions with and without curly braces (blocks vs no blocks). so that was one of the fixes.

both here and below, `align.preset = most` is used and it enables multiline alignment. since these methods are not separated by blank lines (those should interrupt alignment), they are aligned.

the difference with the case below is that includes alignment of `:` (and they do not align, hence `=` we don't get to `=`), while here there are only equals.

Reply:
> and they do not align

Is it an issue that there is multiple `:` on the line? We do not try to align in that case?

Reply:
the way I've implemented it _now_ is that parameter colons should only align with parameters from the same definition (that is, the align owner is the definition) whereas the align owner for the equals is the template which contains the definition.

you can only have one align owner for all tokens on a given line, so it's either colons or equals but not both.

might need to adjust the logic to figure out which should be used when both are present. give me some time, please.

Reply:
ptal, added a commit to choose the largest align container.